src/test/config.sol

```diff
@@ -95,8 +95,8 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x16787F838D6562D7A0337ff65130b6a9a1255C77),         // populate with deployed spell if deployed
-            deployed_spell_created:         1668284243,         // use `./scripts/get-created-timestamp.sh <deployment-tx>`
+            deployed_spell:                 address(0),         // populate with deployed spell if deployed
+            deployed_spell_created:         0,                  // use `./scripts/get-created-timestamp.sh <deployment-tx>`
             previous_spell:                 address(0),         // supply if there is a need to test prior to its cast() function being called on-chain.
             office_hours_enabled:           false,              // true if officehours is expected to be enabled in the spell
```

Comment:
I think OHs should be turned on, following the `RETH-A` changes that will enable the ilk